pages/services/beta-dcos-monitoring/0.4.3-beta/operations/grafana/storage/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: layout.pug
+navigationTitle: Grafana Storage
+title: Grafana Storage
+menuWeight: 20
+excerpt: Grafana Storage
+render: mustache
+model: ../../../data.yml
+---
+
+#include /services/include/beta-software-warning.tmpl
+
+# Configuring Grafana Storage
+
+## DSS Volume
+
+You can configure Grafana to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/) (DSS).
+
+At least one volume must be available for Grafana to use. For more on creating volumes, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-volume/dcos-storage-volume-create).
+
+When configuring Grafana, select `MOUNT` for its volume `type`, choose the minimum `size` for the volume in MB, and enter the `profile` for the volume. For more on volume profiles, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-profile/).
+
+The following configuration will deploy Grafana with a `fast` DSS volume that is 512 MB or larger. If there is no such volume available, Grafana will fail to deploy.
```

Comment:
"The following configuration will deploy Grafana with a fast DSS volume that is 512 MB or larger." change to: "The following configuration will deploy Grafana to a volume that has a volume profile of fast which is 512MB or larger."